client/main.js

```diff
@@ -56,22 +56,22 @@ const getRouteClass = async ( className ) => {
 	let module = null;
 	switch ( className) {
 		case 'wc-connect-create-shipping-label':
-			module = await import(/* webpackChunkName: "shipping-label" */ './apps/shipping-label');
```

Comment:
Removed Magic Comment `webpackChunkName` because we no longer want to bundle them into `[name]`.